canonical-web-and-design/microk8s.io
# Initial rebuild of microk8s homepage for v1

## Done

- A complete update of microk8s.io homepage

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8027/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the [copy doc](https://docs.google.com/document/d/1ObWmhRc4GCIhrBi8gT7yIIs0J_dn_8ravfLtcwhr3uQ/edit#) and install docs: [Linux](https://docs.google.com/document/d/1n2SygKcBgVtpA-HtBWH-nwz31b1uHgIxVzLZtGxK048/edit), https://docs.google.com/document/d/18d2H3uf5HFbaO5sr5COP93xnRxbGwMRlDcTKfwzSGyw/edit, [macos](https://docs.google.com/document/d/10NAfSyV8vMGOpUg4mrvdjx5fTKyY97UtlCpx2FYJ3AM/edit#)

Fixes #248
